invenio_curations/assets/semantic-ui/js/invenio_curations/deposit/DepositBox.js

```diff
@@ -100,10 +120,23 @@ export class DepositBoxComponent extends React.Component {
     this.loading = false;
   };
 
+  checkShouldFetchCurationRequest = async () => {
+    // Fetch curation request instantly when record was updated from an external component
+    const { lastFetchedAt } = this.state;
+    const { lastFormikUpdatedAt } = this.props;
+
+    if (lastFetchedAt < lastFormikUpdatedAt) {
+      this.resetInterval();
+      this.fetchCurationRequest();
+    }
+  };
+
   render() {
     const { latestRequest } = this.state;
     const { record, permissions, groupsEnabled } = this.props;
 
+    this.checkShouldFetchCurationRequest();
```

Comment:
does this trigger on keypress in the TinyMCE fields like description, or just on something like focus loss?
if it's on keypress/character input then that may be a bit much?

Reply:
Form updates are only triggered on specific events like selecting a community or saving a record. After these events, the props will be updated and the timestamp will be set. Then the request will be fetched. So we are good here :)

Reply:
A more appropriate solution would be to check the `record.updated` field, which is usually set on the response from the back end. However, this does not seem to get updated on the local record, as it is not de-serialized from the response.

Created a PR to address this: https://github.com/inveniosoftware/invenio-rdm-records/pull/1838
I would also prefer to use the `updated` field as it allows to get rid of the additional prop.